### ISSUE TITLE
Regforma: Accomodations are not sortable

### ISSUE THREAD

**Issue by bpedersen2** [indico/indico#3322]
In the current accomodation form, the 'No accomodation' option is always  the first and there is no option to sort it.  

As the special no accommodation entry has no indication that it is special (besides the missing fields for
billable etc.),  it happend that the title has been changed to be  main accomodation...

**Comment by ThiefMaster** [comment]
"No accommodation" being the first option is intended, and it **should not** be used for anything else - it has special behavior such as disabling the dates when registering.

**Comment by bpedersen2** [comment]
Yes, the special behaviour is fine, but my users wanted to have "No accommodation" last in the displayed list.
